### PR TITLE
[12.0][MIG] mrp_bom_structure_xlsx: Migration to 12.0

### PR DESCRIPTION
**Pull request by bodedra**

### CODE REVIEW

File: mrp_bom_structure_xlsx/readme/CREDITS.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+* Eficent
```
Comment (tbaden) [inline mrp_bom_structure_xlsx/readme/CREDITS.rst]: Eficent is already Author and Conrtibuter. No need for Extra Credits

> [ This file is optional and contains additional credits, other than
> authors, contributors, and maintainers. ]

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
I did a code review: LGTM, just a minor detail:

**Comment by bodedra** [comment]
@tbaden Attended comment! Thanks for your review.

**Comment by bodedra** [comment]
@jbeficent @mreficent @lreficent  @aheficent

**Review by AaronHForgeFlow — APPROVED** [review]
Functional test on runbot LGTM

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Comment by MiquelRForgeFlow** [comment]
please drop "DROP THIS! oca_dependency update" commit

**Comment by bodedra** [comment]
@aheficent @mreficent Comment Attended!  Thanks for your review.

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Review by tbaden — APPROVED** [review]

**Comment by bodedra** [comment]
@OCA/manufacturing-maintainers can we merge these.  :wink:

**Pull request merged**